src/packages/media/media/property-editors/media-picker/property-editor-ui-media-picker.element.ts

```diff
@@ -1,20 +1,21 @@
 import type { UmbInputMediaElement } from '../../components/input-media/input-media.element.js';
 import '../../components/input-media/input-media.element.js';
+import type { PropertyValueMap } from '@umbraco-cms/backoffice/external/lit';
 import { html, customElement, property, state } from '@umbraco-cms/backoffice/external/lit';
-import {
-	UmbPropertyValueChangeEvent,
-	type UmbPropertyEditorConfigCollection,
-} from '@umbraco-cms/backoffice/property-editor';
+import type { UmbPropertyEditorConfigCollection } from '@umbraco-cms/backoffice/property-editor';
 import type { UmbPropertyEditorUiElement } from '@umbraco-cms/backoffice/extension-registry';
 import { UmbLitElement } from '@umbraco-cms/backoffice/lit-element';
+import { UmbId } from '@umbraco-cms/backoffice/id';
 
 /**
  * @element umb-property-editor-ui-media-picker
  */
 @customElement('umb-property-editor-ui-media-picker')
 export class UmbPropertyEditorUIMediaPickerElement extends UmbLitElement implements UmbPropertyEditorUiElement {
-	@property()
-	public value?: string;
+	@property({ attribute: false })
+	public value: Array<{ key: string; mediaKey: string; mediaTypeAlias: string }> = [];
```

Comment:
Please consider how the migrator converts values from V13 to V14 here. I could fear that it converts these values to a CSV format `key1,key2,key3`. Please verify this before changing the type here.